### PR TITLE
doc: Remove $ character from code blocks

### LINKED ISSUE #227: Break down command line instructions in to single lines
#### Summary

To make instructions as simple as possible, let's break apart copy and paste command line instructions, like so: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/#start-mongodb

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Closes #227 and #200

#### Changes
- Removed $ character from command blocks to make it easier for people to copy commands
-
- 

#### Notes for Reviewers
WiP

#### Checklist
- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by nejraselimovic** [comment]
@benolayinka can you please check this out and provide your feedback? I don't feel like bugging other people with this before you approve

File: doc/content/devices/adding-devices/_index.md [hunk 1]
```diff
@@ -235,21 +256,31 @@ The end device should now be able to join the private network.
...
+```
+
 **LoRaWAN 1.0.x:**
 
 It is also possible to register an ABP activated device using the `--abp` flag as follows:
 
 ```bash
-$ ttn-lw-cli end-devices create app1 dev2 \
-  --frequency-plan-id EU_863_870 \
+ttn-lw-cli end-devices create $APP_ID $DEVICE_ID \
```
Comment (benolayinka) [inline doc/content/devices/adding-devices/_index.md:272]: APP_ID and DEVICE_ID aren't defined

File: doc/content/devices/device-claiming/make-device-claimable.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ To disallow claiming, click **Delete claim authentication code**.
 To configure claiming using the CLI, use the following command:
 
 ```bash
-$ ttn-lw-cli end-device set <app-id> <device-id> --claim-authentication-code.value ABCD \
+ttn-lw-cli end-device set <app-id> <device-id> --claim-authentication-code.value ABCD \
```
Comment (benolayinka) [inline doc/content/devices/device-claiming/make-device-claimable.md:71]: No variables here?

File: doc/content/devices/end-device-templates/mapping.md [hunk 3]
```diff
@@ -21,19 +21,36 @@ This example shows creating a generic device profile that is mapped with provisi
 
 This example uses a **Microchip ATECC608A-MAHTN-T Manifest File**. This file contains provisioning data for The Things Join Server. You can download the example file [here](../microchip-atecc608a-mahtn-t-example.json).
 
+We define some user parameters that will be used below:
+
+```bash
+APP_ID="tes-app"
```
Comment (benolayinka) [inline doc/content/devices/end-device-templates/mapping.md:27]: test-app

File: doc/content/gateways/adding-gateways/_index.md [hunk 4]
```diff
@@ -96,9 +107,9 @@ Some gateways require an API Key with Link Gateway Rights to be able to connect
 Create an API key for the gateway:
 
 ```bash
-$ ttn-lw-cli gateways api-keys create \
-  --name link \
-  --gateway-id gtw1 \
+ttn-lw-cli gateways api-keys create \
+  --name <api-key-name> \
```
Comment (benolayinka) [inline doc/content/gateways/adding-gateways/_index.md:111]: why no variable for api-key-name?

File: doc/content/gateways/adding-gateways/_index.md [hunk 5]
```diff
@@ -111,11 +122,10 @@ Once you have added your gateway to {{% tts %}}, you can also set the locations
 Add an antenna and set its location  with:
 
 ```bash
-$ GTW_ID="your-gateway-id"
-$ ttn-lw-cli gateways set $GTW_ID \
-  --antenna.location.latitude 43.84 \
-  --antenna.location.longitude 18.32 \
-  --antenna.location.altitude 500 \
+ttn-lw-cli gateways set $GTW_ID \
+  --antenna.location.latitude <latitude> \
```
Comment (benolayinka) [inline doc/content/gateways/adding-gateways/_index.md:126]: why no variables for lat and long?

File: doc/content/gateways/lora-basics-station/cups.md [hunk 6]
```diff
@@ -62,10 +62,10 @@ Press **Save Changes** to update the gateway settings. When your gateway connect
 We need to configure CUPS in {{% tts %}} to transmit the LNS API key when a gateway connects. Use the following command to do so, replacing `"your-gateway-id"` with your gateway ID in {{% tts %}} and  `"your-lns-api-key"` with the LNS API key you created in the last step:
 
 ```bash
-$ export GTW_ID="your-gateway-id"
-$ export LNS_KEY="your-lns-api-key"
-$ export SECRET=$(echo -n $LNS_KEY | xxd -ps -u -c 8192)
-$ ttn-lw-cli gateways update $GTW_ID --lbs-lns-secret.value $SECRET
+export GTW_ID="your-gateway-id"
```
Comment (benolayinka) [inline doc/content/gateways/lora-basics-station/cups.md:65]: no need to `export` here actually

File: doc/content/gateways/lora-basics-station/cups.md [hunk 7]
```diff
@@ -120,8 +120,8 @@ This is a file which {{% tts %}} uses to verify the identity of your gateway.
 Use the following commands to create a file called `cups.key`, replacing `"your-cups-api-key"` with the CUPS API key you created above.
 
 ```bash
-$ export CUPS_KEY="your-cups-api-key"
-$ echo "Authorization: Bearer $CUPS_KEY" | perl -p -e 's/\r\n|\n|\r/\r\n/g'  > cups.key
+export CUPS_KEY="your-cups-api-key"
```
Comment (benolayinka) [inline doc/content/gateways/lora-basics-station/cups.md:123]: also no need to export here

File: doc/content/gateways/lora-basics-station/lns.md [hunk 8]
```diff
@@ -54,8 +54,8 @@ This is a file which {{% tts %}} uses to verify the identity of your gateway.
 Use the following command to create a file called `lns.key`, replacing `"your-lns-api-key"` with the LNS API key you created above.
 
 ```bash
-$ export LNS_KEY="your-lns-api-key"
-$ echo "Authorization: Bearer $LNS_KEY" | perl -p -e 's/\r\n|\n|\r/\r\n/g'  > lns.key
+export LNS_KEY="your-lns-api-key"
```
Comment (benolayinka) [inline doc/content/gateways/lora-basics-station/lns.md:57]: ditto

File: doc/content/gateways/troubleshooting/_index.md [hunk 9]
```diff
@@ -32,8 +32,8 @@ See the [Root Certificates Reference]({{< ref "/reference/root-certificates" >}}
 Use the following commands to generate a `api.key` file which is correctly formatted ({{% lbs %}} requires that `.key` files end with a CRLF character).
 
 ```bash
-$ export API_KEY="your-api-key"
-$ echo "Authorization: Bearer $API_KEY" | perl -p -e 's/\r\n|\n|\r/\r\n/g'  > api.key
+export API_KEY="your-api-key"
```
Comment (benolayinka) [inline doc/content/gateways/troubleshooting/_index.md:35]: no need to export

File: doc/content/reference/api/gateway_server_mqtt.md [hunk 10]
```diff
@@ -48,9 +48,9 @@ To forward uplink traffic to the Gateway Server, the MQTT client must publish a
 Below is an example that connects to the Gateway Server as `$GATEWAY_ID`, publishes an uplink message (Protocol Buffer stored as binary file `test-uplink-message`) and disconnects:
 
 ```bash
-$ export GATEWAY_ID="test-gtw@my-tenant"
-$ export GATEWAY_API_KEY="NNSXS.VEEBURF3KR77ZR..." # API key with RIGHT_GATEWAY_LINK rights
-$ mosquitto_pub \
+export GATEWAY_ID="test-gtw@my-tenant"
```
Comment (benolayinka) [inline doc/content/reference/api/gateway_server_mqtt.md:51]: no need to export here

File: doc/content/reference/api/gateway_server_mqtt.md [hunk 11]
```diff
@@ -103,9 +103,9 @@ $ mosquitto_pub \
 To forward a `TxAck` packet to the Gateway Server, the MQTT client must publish a Protocol Buffer of type `ttnpb.TxAcknowledgement` under the topic `v3/<gateway-id>@<tenant-id>/down/ack`.
 
 ```bash
-$ export GATEWAY_ID="test-gtw@my-tenant"
-$ export GATEWAY_API_KEY="NNSXS.VEEBURF3KR77ZR..." # API key with RIGHT_GATEWAY_LINK rights
-$ mosquitto_pub \
+export GATEWAY_ID="test-gtw@my-tenant"
```
Comment (benolayinka) [inline doc/content/reference/api/gateway_server_mqtt.md:106]: no exports in this whole file

**Review by benolayinka — APPROVED** [review]
This is awesome, really nice work Nejra. I know it was a lot!

**Comment by nejraselimovic** [comment]
> This is awesome, really nice work Nejra. I know it was a lot!

Thanks for a great feedback @benolayinka! I fixed everything you proposed. I think others don't really have to review so I'll just merge this.

**Pull request merged**